src/sentry_command.cr

```diff
@@ -0,0 +1,103 @@
+require "yaml"
+require "cli"
+
+require "./sentry"
+
+module Sentry
+
+  class SentryCommand < Cli::Command
```

Comment:
I believe it is a Crystal standard to capitalize acronyms (e.g. CLI, HTTP, JSON)